portfolio/src/App.jsx

```diff
@@ -1,157 +1,145 @@
+          <h1 className="font-bold text-xl">Joshua's Portfolio</h1>
           <ul className="flex gap-4">
-            <li>
-              <a className="text-gray-400 hover:text-white cursor-pointer" a href="#Projects">
-                {" "}
-                Projects
-              </a>
-            </li>
-            <li>
-              <a className="text-gray-400 hover:text-white cursor-pointer" a href="#Experience">
-                {" "}
-                Experience
-              </a>
-            </li>
-            <li>
-              <a className="text-gray-400 hover:text-white cursor-pointer" a href="#About-me">
-                {" "}
-                About Me
-              </a>
-            </li>
-            <li>
-              <a className="text-gray-400 hover:text-white cursor-pointer" a href="#Contact">
-                {" "}
-                Contact
-              </a>
-            </li>
+            {headerOptions.map(({ href, name }) => (
+              <li key={name}>
+                <a
+                  className="text-gray-400 hover:text-white cursor-pointer"
+                  href={href}
+                >
+                  {name}
+                </a>
+              </li>
+            ))}
           </ul>
         </div>
       </header>
       <main>
-        {/*Intro Banner Section */}
         <section>
-        <div className="container m-auto px-4 py-32 flex gap-6">
-          <div>
-            <h2 className="font-bold text-4xl">Hey There! I'm Joshua King, </h2>
-            <h2 className="font-bold text-4xl mt-1 gradiant-text">A Junior Web Developer</h2>
-            <p className="mt-4 text-gray-400">Former general laborer, seeking to apply competent development skills with focus on collaboration, communication and passion.</p>
-          </div>
-          <div className="relative">
-            <img src={Porfile} width={420} className="relative z-10" />
+          <div className="container m-auto px-4 py-32 flex gap-6 items-center">
+            <div>
+              <h2 className="font-bold text-4xl">
+                Hey There! I'm Joshua King,
+              </h2>
+              <h2 className="font-bold text-4xl mt-1 gradient-text">
+                A Junior Web Developer
+              </h2>
+              <p className="mt-4 text-gray-400">
+                Former general laborer, seeking to apply competent development
+                skills with focus on collaboration, communication and passion.
+              </p>
+            </div>
+            <div className="relative">
+              <img src={Profile} width={420} className="relative z-10" />
+            </div>
           </div>
-        </div>
         </section>
-        {/*Projects section */}
-        <section className="py-10" section id="Projects" >
-          <div className="container m-auto px-4 sm:py-12">
+
+        <section className="py-10" section id="projects">
```

Comment:
Went ahead and updated the `id` here to be lowercase. It's pretty standard practice to use all lowercase in the URL. So since this would thus get populated in the URL if this section was ever navigated to from the respective option in the header, I figured I would just suggest making it lowercase